src/systemd/agent/ossec-execd.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=OSSEC Execd
 PartOf=ossec-agent.target
+Before=ossec-ossec-syscheckd.service ossec-agentd.service ossec-logcollector.service
```

Comment:
typo here ? should be ossec-syscheckd

Reply:
Thanks for spotting those mistakes. They should be corrected now.